src/script.js

```diff
@@ -1,21 +1,27 @@
 import { EventKey } from "./consts";
 import handleFormSubmit from "./handleFormSubmit";
 
-const attachEventListeners = () => {
-  const inputElement = document.getElementById("input");
-  const buttonElement = document.getElementById("button");
+let inputElement, buttonElement, amountFoundElement;
+
+const initilizeElements = () => {
```

Comment:
Are we really initialising elements here? or just fetching a reference?
I think we should rethink this naming here.

```suggestion
const initilizeElementsRefrences = () => {
```